board/docker-compose.yml

```
@@ -1,6 +1,8 @@
 services:
   board:
-    image: k8s-study/board:1.1
+    image: k8s-study/board:1.0
     container_name: board
+#    environment:
+#      server.port: 8081
```

`board/docker-compose.yml` 파일은 이번 과제의 수정 대상이 아닙니다. 해당 파일을 롤백 해주세요
```suggestion
    image: k8s-study/board:1.1
    container_name: board
```